### ISSUE TITLE
Locations are missing in the tag page

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kofil/moloko#4]
```
Moloko 0.9.2b on HTC Desire running stock Froyo.

On the RTM website, the tag cloud contains tags and locations. Moloko only 
displays tags. 

For example, I have a location "@home" that is set to tasks to be done at home.
```

Original issue reported on code.google.com by `david.ve...@gmail.com` on 21 Nov 2010 at 8:14

**Comment by GoogleCodeExporter** [comment]
Original comment by `ronny.roehricht` on 22 Nov 2010 at 7:01

**Comment by GoogleCodeExporter** [comment]
```
To be done if locations are shown in the task details view.
```

Original comment by `ronny.roehricht` on 22 Nov 2010 at 7:02

**Comment by GoogleCodeExporter** [comment]
Original comment by `ronny.roehricht` on 24 Nov 2010 at 5:49
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Hint: Long click on location in tags view opens location chooser.

Fixed in r168 / 0.9.5b
```

Original comment by `ronny.roehricht` on 4 Dec 2010 at 9:35
- Changed state: **Fixed**
